src/fluxo.object_store.js

```diff
@@ -26,12 +28,42 @@ class ObjectStore {
     this.set(data);
 
     this.registerComputed();
+
+    this.warnMissingAttributes();
+  }
+
+  warnMissingAttributes () {
+    if (!this.constructor.attributes) { return; }
+
+    for (let attributeName in this.constructor.attributes) {
+      if (this.contract(attributeName).required && !this.data[attributeName]) {
+        this.warnMissingAttribute(attributeName);
+      }
+    }
+  }
+
+  warnMissingAttribute (attributeName) {
+    var identifier = "";
+
+    if (this.data.id) {
+      identifier = `id: ${this.data.id}`;
+    } else {
+      identifier = `cid: ${this.cid}`;
+    }
+
+    console.warn(`Missing the required "${attributeName}" attribute on the "${this.constructor.name}" (${identifier})`);
```

Comment:
I think you need to check if `console` and `console.warn` is available, otherwise some browsers may thrown an error and stop the execution if the console isn't available.